src/lib/viewers/image/ImageViewer.js

```diff
@@ -192,6 +197,20 @@ class ImageViewer extends ImageBaseViewer {
             // Since we are taking offsetWidth, we only need to apply the zoom to the width
             // as clearing the height will preserve the aspect ratio
             newWidth = type === 'in' ? width * IMAGE_ZOOM_SCALE : width / IMAGE_ZOOM_SCALE;
+
+            // We want to calculate the zoomedWidth / zoomedHeight to determine if we should toggle annotation mode to None
+            const ratio = Math.min(viewport.width / width, viewport.height / height);
+            const zoomedWidth = newWidth * ratio;
+            const zoomedHeight =
+                type === 'in' ? height * IMAGE_ZOOM_SCALE * ratio : (height / IMAGE_ZOOM_SCALE) * ratio;
+
+            // If the image is overflowing the viewport, we should set annotation mode to be NONE so that the user can pan
+            if (zoomedWidth > viewport.width || zoomedHeight > viewport.height) {
+                if (this.options.enableAnnotationsImageDiscoverability) {
+                    this.processAnnotationModeChange(this.annotationControlsFSM.transition(AnnotationInput.CANCEL));
+                    this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
+                }
+            }
```

Comment:
I'd be inclined to leave this `zoom` method alone and perhaps react to it via event or perhaps here https://github.com/box/box-content-preview/blob/c29dddc59d652cd597fd518a8af5bb78bdd5f16c/src/lib/viewers/image/ImageBaseViewer.js#L95

Reply:
Sounds good, I've moved this logic up to the ImageBaseViewer class. I also had to move a few helper methods as well because it seems to make more sense for the child to call the parent helpers rather than the parent calling the child methods.